src/SDKs/SqlManagement/Sql.Tests/DataMaskingScenarioTests.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using Microsoft.Azure.Management.Sql.Models;
+using Microsoft.Azure.Management.ResourceManager;
+using Microsoft.Azure.Management.Sql;
+using Xunit;
+using Microsoft.Azure.Test.HttpRecorder;
+
+namespace Sql.Tests
+{
+    public class DataMaskingScenarioTests
+    {
+        [Fact]
+        public void TestCreateUpdateGetDataMaskingRules()
+        {
+            string testPrefix = "sqlcrudtest-";
+            string suiteName = this.GetType().FullName;
+            SqlManagementTestUtilities.RunTestInNewV12Server(suiteName, "TestCreateUpdateGetDataMaskingRules", testPrefix, (resClient, sqlClient, resourceGroup, server) =>
```

Comment:
Can you use new SqlManagementTestContext instead? See Restore LTR test for example